client/components/CustomBankField.js

```diff
@@ -0,0 +1,74 @@
+import {has, translate as t} from '../Helpers';
+import T from './Translated';
+
+export default class CustomBankField extends React.Component {
+
+    constructor(props) {
+        has(props, 'params');
+
+        super(props);
+    }
+
+    getValue() {
+        let node = this.refs[this.props.params.name].getDOMNode();
```

Comment:
No need that the ref be actually dynamic: you could have the two fields in `render()` have for ref value `field`, and use `this.refs.field.getDOMNode()` here.